src/Common/Lists/Country.php

```diff
@@ -25,254 +294,50 @@ class Country {
-				'SA' => 'Saudi Arabia',
-				'SG' => 'Singapore',
-				'KR' => 'South Korea',
-				'LK' => 'Sri Lanka',
-				'SY' => 'Syria',
-				'TW' => 'Taiwan',
-				'TJ' => 'Tajikistan',
-				'TH' => 'Thailand',
-				'TM' => 'Turkmenistan',
-				'AE' => 'United Arab Emirates',
-				'UZ' => 'Uzbekistan',
-				'VN' => 'Vietnam',
-				'YE' => 'Yemen',
-			],
-			'Oceania'    => [
-				'AS' => 'American Samoa',
-				'AU' => 'Australia',
-				'CK' => 'Cook Islands',
-				'FJ' => 'Fiji',
-				'PF' => 'French Polynesia',
-				'GU' => 'Guam',
-				'KI' => 'Kiribati',
-				'MH' => 'Marshall Islands',
-				'FM' => 'Micronesia',
-				'NR' => 'Nauru',
-				'NC' => 'New Caledonia',
-				'NZ' => 'New Zealand',
-				'NU' => 'Niue',
-				'NF' => 'Norfolk Island',
-				'MP' => 'Northern Mariana Islands',
-				'PW' => 'Palau',
-				'PG' => 'Papua New Guinea',
-				'PN' => 'Pitcairn Islands',
-				'WS' => 'Samoa',
-				'SB' => 'Solomon Islands',
-				'TK' => 'Tokelau',
-				'TO' => 'Tonga',
-				'TV' => 'Tuvalu',
-				'UM' => 'U.S. Minor Outlying Islands',
-				'VU' => 'Vanuatu',
-				'WF' => 'Wallis and Futuna',
-			],
-			'Europe'     => [
-				'AX' => 'Åland Islands',
-				'AL' => 'Albania',
-				'AD' => 'Andorra',
-				'AT' => 'Austria',
-				'BY' => 'Belarus',
-				'BE' => 'Belgium',
-				'BA' => 'Bosnia and Herzegovina',
-				'BG' => 'Bulgaria',
-				'HR' => 'Croatia',
-				'CY' => 'Cyprus',
-				'CZ' => 'Czech Republic',
-				'DK' => 'Denmark',
-				'EE' => 'Estonia',
-				'FO' => 'Faroe Islands',
-				'FI' => 'Finland',
-				'FR' => 'France',
-				'DE' => 'Germany',
-				'GI' => 'Gibraltar',
-				'GR' => 'Greece',
-				'GG' => 'Guernsey',
-				'HU' => 'Hungary',
-				'IS' => 'Iceland',
-				'IE' => 'Ireland',
-				'IM' => 'Isle of Man',
-				'IT' => 'Italy',
-				'JE' => 'Jersey',
-				'LV' => 'Latvia',
-				'LI' => 'Liechtenstein',
-				'LT' => 'Lithuania',
-				'LU' => 'Luxembourg',
-				'MT' => 'Malta',
-				'MD' => 'Moldova',
-				'MC' => 'Monaco',
-				'ME' => 'Montenegro',
-				'NL' => 'Netherlands',
-				'MK' => 'North Macedonia',
-				'NO' => 'Norway',
-				'PL' => 'Poland',
-				'PT' => 'Portugal',
-				'RO' => 'Romania',
-				'RU' => 'Russia',
-				'SM' => 'San Marino',
-				'RS' => 'Serbia',
-				'SK' => 'Slovakia',
-				'SI' => 'Slovenia',
-				'ES' => 'Spain',
-				'SJ' => 'Svalbard and Jan Mayen',
-				'SE' => 'Sweden',
-				'CH' => 'Switzerland',
-				'UA' => 'Ukraine',
-				'GB' => 'United Kingdom',
-				'VA' => 'Vatican City',
-			],
-		];
-
 		/**
 		 * Filter the list of countries.
 		 *
 		 * @since TBD
 		 *
 		 * @param array $countries The list of countries. Grouped by continent/region.
 		 */
-		return apply_filters( 'tec_country_list', $countries );
+		return apply_filters( 'tec_common_country_list', self::COUNTRIES );
+	}
+
+	/**
+	 * Get a formatted list of countries.
+	 *
+	 * @since TBD
+	 *
+	 * @return array<string,array<string,mixed>> The formatted list of countries.
+	 */
+	public function get_country_list_formatted(): array {
```

Comment:
This is not "formatting" - this is adding data.
The functionality is great, but the function name is misleading.

Reply:
"with additional data"
"with gateway info"
something that tells folks it's not just "making it pretty" but adding information/data